src/components/EventList/index.jsx

```diff
@@ -62,7 +64,13 @@ const EventList = ({ events }) => {
 
   return (
     <Card>
-      <CardHeader title={header} className={styles.header} />
+      <CardHeader
+        title={header}
+        className={styles.header}
+        style={{
+          backgroundColor: theme.colorSchemes.light.palette.primary[300],
+        }}
```

Comment:
This is not exactly what we want. The end goal is that, when the MUI Theme is updated (e.g. because the user selected Dark Mode), every component that uses theme colors will change styles.

What you have currently here will not update when the MUI Theme is updated, it will only change when the code in the `src/theme.ts` file is updated (which obviously cannot be done by user input). Notice, for example, that you've hard-coded `theme.colorSchemes.light` here - so what would happen when the UI switches to using `theme.colorSchemes.dark` instead?

There are a couple of correct ways to do this. The simplest, for our current arrangement, is to use the new `--mui-*` CSS variables to style this component via CSS (similar to how it was previously being styled by the `styles.header` class).

The other way, if we want to put the styling info into the JSX file (which is debatable) is to use the `theme` variable injected by MUI (via the Provider at the root of the component tree). That is, instead of using the static `theme` object that we created in `src/theme.ts` but that will never be updated, we want to use the dynamic `theme` variable that MUI is managing and that can be changed by, for example, a Dark Mode toggle.

There are basically two ways to access this dynamic `theme` variable:
1. Using the `sx` prop of MUI components, as in this example: https://mui.com/system/palette/#color
2. With the `useTheme` hook: https://mui.com/material-ui/customization/theming/#accessing-the-theme-in-a-component